src/builder.rs

```diff
@@ -53,14 +49,16 @@
 //! let interface = display_interface_spi::SPIInterfaceNoCS::new(spi, dc);
 //! let display: TerminalMode<_> = Builder::new().connect(interface).into();
 //! ```
+//!
+//! [`I2CDIBuilder`]: crate::builder::I2CDIBuilder
+//! [`DisplayProperties`]: crate::properties::DisplayProperties
+//! [`GraphicsMode`]: crate::mode::graphics::GraphicsMode
+//! [`TerminalMode`]: crate::mode::terminal::TerminalMode
```

Comment:
Much cleaner, thanks. As mentioned in the review comments, please change these links (and all others in the PR) back to relative paths. Linkchecker and manual docs surfing should help here.